src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestDetailsView.tsx

```diff
@@ -146,12 +147,14 @@ const ChangeRequestDetailsView: React.FC<ChangeRequestDetailsProps> = ({
         changes={changeRequest.implementedChanges || []}
         overallDateImplemented={changeRequest.dateImplemented}
       />
+
       {reviewModalShow && (
         <ReviewChangeRequest modalShow={reviewModalShow} handleClose={handleReviewClose} cr={changeRequest} />
       )}
       {deleteModalShow && (
         <DeleteChangeRequest modalShow={deleteModalShow} handleClose={handleDeleteClose} cr={changeRequest} />
       )}
```

Comment:
After running my local app, I've seen one potential issue with locating the popup tab as the fixed position at the bottom of the CR details page. It is hard for the users to click on `Implemented Changes` dropdown as shown in the screenshot b/c the space between the popup tab and its dropdown is small. 

What do we think bout this? I think we can fix this issue by adding the blank space before the popup tab line, so the user can scroll way down to see a full `Implemented Changes` dropdown section. 

<img width="960" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/91857946/7e4c20c3-5d79-4e97-bc96-b8d4244970c1">

Reply:
As addressed in the Slack thread with Sharon, this issue will be resolved in #1639 since the details page redesign is getting rid of the dropdowns.